core/src/autogluon/core/models/abstract/abstract_model.py

```diff
@@ -79,9 +76,7 @@ def __init__(self,
                  name: str = None,
                  problem_type: str = None,
                  eval_metric: Union[str, metrics.Scorer] = None,
-                 hyperparameters=None,
-                 quantile_levels=None,
```

Comment:
I'm surprised removing this doesn't break quantile regression?

Or perhaps it is longer being properly passed into the model-constructor here: 
https://github.com/awslabs/autogluon/blob/70d64be1419e69e81210d162ffc0827ee7cfc3ea/tabular/src/autogluon/tabular/trainer/model_presets/presets.py#L319-L320

Reply:
Nevermind was looking at wrong branch, I now see where ag_args is updated in trainer